File: Latihan 1.2/Php/Sepakbola.php
<?php

class DaftarSepakbola
{	
	// inisialisasi atribut private 
	
	private $nama = "";
	private $negara = "";
	private $tahun = 0;
	private $pemain = 0;

	// construksi atribut
	public function __construct($nama = "", $negara = "", $tahun = 0, $pemain = 0)
	{
		$this->nama = $nama;
		$this->negara = $negara;
		$this->tahun = $tahun;
		$this->pemain = $pemain;
	}
	
	public function setNama($nama){
		$this->nama = $nama;
	}

	public function getNama(){
		return $this->nama;
	}

	public function setNegara($negara){
		$this->negara = $negara;
	}

	public function getNegara(){
		return $this->negara;
	}

	public function setTahun($tahun){
		$this->tahun = $tahun;
	}

	public function getTahun(){
		return $this->tahun;
	}

	public function setPemain($pemain){
		$this->pemain = $pemain;
	}

	public function getPemain(){
		return $this->pemain;
	}

}

?>

File: Latihan 1.2/Phyton/Sepakbola.py
class Sepakbola():

	# nama Atribut
	__nama = "blank"
	__negara = "blank"
	__tahun = 0
	__pemain= 0

	# constructor
	def __init__(self, nama = "blank", negara = "blank", tahun = 0, pemain = 0):
		self.__nama = nama
		self.__negara = negara
		self.__tahun = tahun
		self.__pemain = pemain

	#setter
	def setNama(self,nama):
		self.__nama = nama
	def setNegara(self,negara):
		self.__negara = negara
	def setTahun(self,tahun):
		self.__tahun = tahun
	def setPemain(self,pemain):
		self.__pemain = pemain

	#getter
	def getNama(self):
		return self.__nama
	def getNegara(self):
		return self.__negara
	def getTahun(self):
		return self.__tahun
	def getPemain(self):
		return self.__pemain
